.drone.yml

```diff
@@ -187,27 +179,15 @@ steps:
           - feature/*
       event: pull_request
 
-  - name: build_acceptance_branch
-    pull: if-not-exists
-    image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/dind
-    commands:
-    - n=0; while [ "$n" -lt 60 ] && [ ! docker stats --no-stream ]; do n=$(( n + 1 )); sleep 1; done
-    - docker build -f Dockerfile-acceptance -t acceptance-$${DRONE_COMMIT_SHA} .
-    when:
-      branch: master
-      event: pull_request
-
   - name: test_acceptance_branch
-    pull: if-not-exists
-    image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/dind
+    <<: *acceptance_tests
```

Comment:
Changed old acceptance test steps (so-acceptance, uses docker) to new Playwright way